README.md

```diff
@@ -95,6 +95,7 @@ Consider submitting to [the deno.land/x](https://github.com/denoland/registry/bl
 # Tools
 
 - [clone](https://github.com/ekaragodin/clone) - a simple utility for the convenient clone.
+- [darn](https://github.com/denoserverless/darn) = Kind of like Yarn, but for Deno
```

Comment:
Please use same format.
Also i would prefer `Package manager. eg: yarn`

Reply:
Sorry, what format? Oh, I accidentally wrote `=`?

As about the other thing, I just wrote what it describes itself.